.env.example

```diff
@@ -1,3 +1,6 @@
 # We need to set both the username and password as environment variables in order to use the app
 GITHUB_ANALYZER_USERNAME=postgres
-GITHUB_ANALYZER_PASSWORD=postgres
\ No newline at end of file
+GITHUB_ANALYZER_PASSWORD=postgres
+
+# Must be set as the webhook secret on your github app
+GITHUB_ANALYZER_WEBHOOK_SECRET=78e59601e29b4db895f86fe6ac4442da9a6b8843
```

Comment:
🤔 don't commit actual keys. you can have `fake-key` as the value in the `.env.example`